src/app/model/migrations/migrations/to0002/MigrationDtoTypes0002.ts

```diff
@@ -0,0 +1,28 @@
+import { ImageUrl } from '@app/shared/lib';
+
+export type ActivityRecordDto = {
```

Comment:
You'll not need these types after you change logic to take activityName from FlowState.pipeline. Clean up.